week-3/Homework/mandatory/2-bush-berries.js

```diff
@@ -10,8 +10,13 @@
   Use the tests to confirm which message to return
 */
 
-function bushChecker() {
-
+function bushChecker(arr) {
+  let allTheSame = arr.every(element => element === "pink")
+  if (allTheSame === true) {
```

Comment:
you can use if statement in easy way like this:
if (allTheSame){
}

Reply:
or you can do it all in one nice line 

 return  colours.every(colour => colour === "pink") ? "Bush is safe to eat from" : "Toxic! Leave bush alone!"